pkgs/development/python-modules/aplpy/default.nix

```diff
@@ -27,13 +27,24 @@ buildPythonPackage rec {
     sha256 = "239f3d83635ca4251536aeb577df7c60df77fc4d658097b92094719739aec3f3";
   };
 
-  patches = [ (fetchpatch {
+  patches = [
+    (fetchpatch {
       # Can be removed in next release after 2.0.3
```

Comment:
?

Surprised this is even applying then. Though it is excluding a lot of files..

Reply:
Yeah, it seems like they reworked all of their CI stuff between the latest release to pypi and this patch, so changes to those files needed to be skipped.

Reply:
But what happens if we remove this patch? It would be nice to at least change the confusing comment...

Reply:
The patch is still necessary. It's from https://github.com/NixOS/nixpkgs/pull/107770. I apologize for the confusing comment, I'll update it.